build/zip.py

```diff
@@ -25,6 +25,13 @@
   # //chrome/browser/resources/ssl/ssl_error_assistant, but we don't need to
   # ship it.
   'pyproto',
+
+  # On Windows, this binary doesn't exist (the crashpad handler is built-in).
+  # On MacOS, the binary is called 'chrome_crashpad_handler' and is inside the
+  # app bundle.
+  # On Linux, we don't use crashpad, but this binary is still built for some
+  # reason. Exclude it from the zip.
+  'crashpad_handler',
```

Comment:
`./crashpad_handler`

> This binary was being built & included on Linux despite the fact that we do not use crashpad on Linux.

`gn refs out/Testing //third_party/crashpad/crashpad/handler:crashpad_handler` shows
https://source.chromium.org/chromium/chromium/src/+/master:components/crash/core/app/BUILD.gn;l=131

Reply:
Yep, upstream is experimenting with crashpad on linux (they have it enabled on Android). We aren't though.